draft-ietf-rats-corim.md

```diff
@@ -768,68 +769,79 @@ The types defined for a group identified are UUID and variable-length opaque byt
 {::include cddl/group-id-type-choice.cddl}
 ~~~
 
-##### Measurements
+##### Properties
+
+A Property is a key-value pair that in CoRIM represents a single piece of actual state or a possible state.
+
+A Property in CoRIM always appears within a triple.
+The triple expresses the intended meaning of the Property.
+In all cases, a Property is made by an authority, such as an Attester, Endorser, or Reference Value Provider.
+
+A Property from an Attester is called Evidence.
+A Property in a `reference-triple-record` is called a Reference Value.
+A Property in another triple is called an Endorsement.
```

Comment:
Does the list in [5.1.4 Triples](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#section-5.1.4) fit that statement?